packages/dynamic-import-vars/src/index.js

```diff
@@ -52,10 +52,13 @@ function dynamicImportVariables({ include, exclude, warnOnError } = {}) {
             // will turn these into chunks automatically
             ms.prepend(
               `function __variableDynamicImportRuntime${dynamicImportIndex}__(path) {
-   switch (path) {
-  ${paths.map((p) => `   case '${p}': return import('${p}');`).join('\n  ')}
-     default: return Promise.reject(new Error("Unknown variable dynamic import: " + path));
-   }
+  switch (path) {
+${paths.map((p) => `    case '${p}': return import('${p}');`).join('\n')}
+${`    default: return new Promise((resolve, reject) => {
```

Comment:
Should we use a standard function in here vs the arrow one?

Reply:
I suppose an arrow function isn't strictly necessary. Do you see a downside to it in this case? There's no scope it's ignoring.

Reply:
It depends what browsers this is targeting. Promises are polyfillable but the arrow syntax isn’t, so it would be safer to stick with the older function format I think.

Reply:
Ah ok. Didn't realize we were going back that far with browser support, but yeah I guess it's the safest thing to do and no real downside.